gallery/src/pages/AdditionalResources.tsx

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {GalleryTile} from '../gallery-components/GalleryTiles';
+
+const AdditionalResources = () =>
+  <GalleryTile title="Additional Resources">
+    <p className="nx-p">
+      We have curated a list of additional resources you can use to increase your understanding of react and redux.
+    </p>
+
+    <h3 className="nx-h3 nx-tile__section-header">Classes / Tutorials</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://reactjs.org/tutorial/tutorial.html" target="_blank">
+          Official ReactJS Tutorial
+        </a>
+        {' '}(Beginner Level)
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://redux.js.org/basics/basic-tutorial" target="_blank">Redux Getting Started</a>
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://egghead.io/courses/getting-started-with-redux" target="_blank">
+          Getting started with Redux
+        </a>
+        {' '}(Beginner Level)
+      </li>
+    </ul>
+
+    <h3 className="nx-h3 nx-tile__section-header">Good Reads</h3>
+    <ul className="nx-list nx-list--bulleted">
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://reactjs.org/docs/hello-world.html" target="_blank">Main Concepts in React</a>
+      </li>
+      <li className="nx-list__item">
+        <a rel="noreferrer" href="https://reactjs.org/docs/thinking-in-react.html" target="_blank">Thinking in React</a>
+      </li>
+      <li className="nx-list__item">
+        IQ (and many other apps utilizing React and Redux together) heavily utilize the official{' '}
+        <a rel="noreferrer" href="https://react-redux.js.org/using-react-redux/connect-mapstate" target="_blank">
+          react-redux
+        </a>
+        {' '}connectors, as well as the{' '}
+        <a rel="noreferrer" href="https://redux-toolkit.js.org/introduction/quick-start" target="_blank">
+          Redux toolkit
```

Comment:
I had never heard of the redux "toolkit" before, so I think it is inaccurate to say it is used within IQ.

Reply:
fixed; 3ca289d